xen/arch/arm/cpufreq/mailbox.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * Based on Linux drivers/mailbox/mailbox.h
+ * => commit 86c22f8c9a3b71d42d38bfcd80372de72f573713
+ *
+ * No Xen modification.
+ */
+
```

Comment:
This file is not used. Please drop it.

Reply:
Why do you think so? It is commonly used by mailbox infrastructure

Reply:
Yep, my bad. This github squashed the mailbox.c so find did not show this header usage.